docs/sources/trackers/google-tag-manager/ecommerce-tag-template/configuration/index.md

```diff
@@ -0,0 +1,98 @@
+
+**Set Custom Timestamp**
+
+Set this to a UNIX timestamp in case you want to [override the default timestamp](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/index.md#setting-the-true-timestamp) used by Snowplow.
+
+  </TabItem>
+  <TabItem value="ga4" label="GA4 Ecommerce">
+
+![](images/02_ga4_tracking_parameters.png)
+
+#### GA4 Ecommerce Function
+
+In this section you can select the [Google Analytics 4 Ecommerce function](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/ecommerce/index.md) to use.
+
+#### GA4 Ecommerce Arguments
+
+**DataLayer ecommerce**
+
+Here you can specify the dataLayer ecommerce variable to use, i.e. a variable that returns the `ecommerce` object itself.
+
+**Options object**
+
+Here you can specify a variable returning an object holding additional information for the ecommerce event (e.g. including `currency`, `finalCartValue`, `step`, etc).
+
+  </TabItem>
+  <TabItem value="ua" label="Universal Analytics Enhanced Ecommerce">
+
+![](images/02_ua_tracking_parameters.png)
+
+#### Universal Analytics Enhanced Ecommerce Function
+
+In this section you can select the [Universal Analytics Enhanced Ecommerce function](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/ecommerce/index.md) to use.
+
+#### Universal Analytics Enhanced Ecommerce Arguments
+
+**DataLayer ecommerce**
+
+Here you can specify the dataLayer ecommerce variable to use.
+
+**Options object**
+
+Here you can specify a variable returning an object holding additional information for the ecommerce event (e.g.including currency, finalCartValue, step etc).
+
+  </TabItem>
+</Tabs>
+
+## Snowplow Tracker and Ecommerce Plugin Settings
+
+![](images/04_tracker_plugin_settings.png)
+
+### Tracker Settings
+
+The Snowplow Ecommerce tag template **requires** a Snowplow Settings Variable to be setup. In this section you can select the Google Tag Manager variable of type [Snowplow Settings](/docs/sources/trackers/google-tag-manager/quick-start/index.md) to use.
+
+### Plugin Settings
+
+In this section you can select how the plugin will be added. The available options are:
+
+- **jsDelivr**: To get the plugin URL from jsDelivr CDN. Choosing this option allows you to specify the plugin version to be used.
+- **unpkg**: To get the plugin URL from unpkg CDN. Choosing this option allows you to specify the plugin version to be used.
+- **Self-hosted**: To get the plugin library from a specified URL. This option requires a [Permission](https://developers.google.com/tag-platform/tag-manager/templates/permissions) change to allow injecting the plugin script from the specified URL.
+- **Do not add**: To not add the plugin (e.g. when using a [Custom Bundle](/docs/sources/trackers/javascript-trackers/web-tracker/plugins/configuring-tracker-plugins/index.md) with the plugin already included)
```

Comment:
Maybe a link to `/docs/sources/trackers/javascript-trackers/web-tracker/plugins/index.md` advising to check your bundled plugins would be good here since this situation is a bit different between v3/v4.

Reply:
Just to clarify, by _the situation_, are you referring to the new set of default plugins for v4? I've added a bit about that, but not 100% that's what you were after.